assets/css/app.css

```
@@ -1,51 +1,55 @@
 @import "tailwindcss";
 
 @plugin "daisyui" {
-    themes: kira;
+    themes: gemma;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)